### DISCUSSION TITLE
datasets/detect/roboflow-100/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#13904] [Docs]
# datasets/detect/roboflow-100/

Explore the Roboflow 100 dataset featuring 100 diverse datasets designed to test object detection models across various domains, from healthcare to video games.

https://docs.ultralytics.com/datasets/detect/roboflow-100/

**Comment by mariem158** [comment]
les étapes pour télécharger ce dataset roboflow100 pour détection aerienne ?

**Reply by glenn-jocher** [reply]
@mariem158 bonjour!

Pour télécharger et utiliser le dataset Roboflow 100 pour la détection aérienne, suivez ces étapes :

1. **Accédez à Roboflow Universe**:
   - Visitez [Roboflow Universe](https://universe.roboflow.com/roboflow-100).
   - Recherchez les datasets dans la catégorie "Aerial".

2. **Téléchargez le Dataset**:
   - Cliquez sur le dataset aérien de votre choix.
   - Utilisez le bouton "Export" pour télécharger les annotations et les images dans le format souhaité (par exemple, COCO, YOLO).

3. **Utilisez le Dataset avec Ultralytics YOLO**:
   - Assurez-vous d'avoir les dernières versions de `torch` et `ultralytics` installées.
   - Voici un exemple de code pour entraîner un modèle YOLOv8 avec un dataset aérien téléchargé :

    ```python
    from ultralytics import YOLO

    # Charger le modèle YOLOv8
    model = YOLO('yolov8n.pt')

    # Entraîner le modèle avec le dataset aérien
    model.train(data='path/to/aerial_dataset.yaml', epochs=100, imgsz=640)
    ```

4. **Benchmarking**:
   - Vous pouvez également utiliser le script de benchmarking fourni dans la documentation pour évaluer les performances de votre modèle sur différents datasets du Roboflow 100.

Pour plus de détails, consultez la documentation complète sur le dataset Roboflow 100 [ici](https://docs.ultralytics.com/datasets/detect/roboflow-100/).

N'hésitez pas à revenir si vous avez d'autres questions ! 😊

Cordialement,
L'équipe Ultralytics
